recipes/supertree/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set name = "supertree" %}
+{% set version = "0.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0a802b89b9358b437d0b4cdec69ffb8f8babcc16005ffe017464218a915aaa9e
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - pip
+  run:
+    - python >=3.9
+    - numpy
+    - pandas
+    - IPython
+
+test:
+  imports:
+    - supertree
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/mljar/supertree
+  summary: Interactive Decision Tree Visualization
+  license: LicenseRef-LICENSE.txt
+  license_file: LICENSE.txt
```

For the extra files you can do something like this.

```suggestion
  license_file:
    - LICENSE.txt
    - file2
    - file3
```